src/App.jsx

```diff
@@ -1,8 +1,33 @@
-import { useState } from "react";
+import { Routes, Route } from "react-router-dom";
+import { useEffect, useState } from "react";
 import "./App.css";
+import Header from "./components/Header";
+import Nav from "./components/Nav";
+import Signin from "./components/Signin";
+import Home from "./components/Home";
+import ArticleList from "./components/ArticleList";
+import { getCategories } from "./api";
 
 function App() {
-  return <></>;
+  const [category, setCategory] = useState([]);
+  useEffect(() => {
+    getCategories().then(({ data }) => {
+      setCategory(data);
+    });
+  }, []);
```

Comment:
You aren't doing anything with this state currently. Please get rid of it until you need it